packages/mobile/package.json

```diff
@@ -21,12 +21,12 @@
     "build:ios:stage": "xcodebuild -workspace ios/AudiusReactNative.xcworkspace -scheme Staging -sdk iphoneos -configuration AppStoreDistribution archive -archivePath $PWD/build/AudiusReactNative.xcarchive",
     "export:ios": "xcodebuild -exportArchive -archivePath $PWD/build/AudiusReactNative.xcarchive -exportOptionsPlist ios/exportOptions.plist -exportPath $PWD/build -allowProvisioningUpdates",
     "upload:ios": "xcrun altool --upload-app -type ios --file build/AudiusReactNative.ipa --username $APPLE_ID --password $APPLE_ID_PASSWORD",
-    "codepush-android:rc": "appcenter codepush release-react -a Audius-Music/Audius-Android -d Staging",
-    "codepush-android:prod": "appcenter codepush release-react -a Audius-Music/Audius-Android -d Production",
-    "codepush-ios:rc": "appcenter codepush release-react -a Audius-Music/Audius -d Staging",
-    "codepush-ios:prod": "appcenter codepush release-react -a Audius-Music/Audius -d Production",
-    "codepush-history:prod": "appcenter codepush deployment history -a Audius-Music/Audius Production",
-    "codepush-history:rc": "appcenter codepush deployment history -a Audius-Music/Audius Staging",
+    "codepush-ios-history:prod": "appcenter codepush deployment history -a Audius-Music/Audius Production",
```

Comment:
What is `history`?

Reply:
it gives you a list of recent deployments. i thought it might be handy in case we need to do rollbacks or something